testing/internal/e2e/scope/scope_test.go

```diff
@@ -0,0 +1,231 @@
+package scope_test
```

Comment:
Both these tests still need to be rewritten to be independent of previous runs. Will probably run in a temporary scope instead of "global".